vowel_duplicates.py

```diff
@@ -2,17 +2,17 @@
 def vowels(word):
     new_word = []
     vowel = ['a', 'e', 'i', 'o', 'u']
+    #get number of duplicates using set function
+    duplicates = len(word) - len(set(word))
+
     for x in word:
-        if x in vowel:
+        if x in vowel and x not in new_word:
            new_word.append(x)
-           count = new_word.count(x)
-           if count > 1:
-               new_word.pop(1)
-               count = count-1
-               return new_word, count
+        else:
+           continue
 
-    return new_word
+    return new_word, duplicates
```

Comment:
Return new word as a string and not a list. You can convert an array to a string using the `join` method
Forexample
```python
name = name = "".join(["d","e","n","n","i","s"])
print(name)
```
Prints a name you might know from an array of letters

Reply:
Thank you @dennisja , this has simplified my output from (['e', 'a'], 3) to this ('ea', 3)